project1.css

```diff
@@ -0,0 +1,172 @@
+header {
+  position: absolute;
+  width: 1440px;
+  height: 846px;
+  left: 200px;
+  top: 71px;
+}
```

Comment:
- [ ] the content should be aligned using flexbox, right now when opening the application in mobile view it is distorted and doesn't look like the template, have a look at the below example to see what it looks like right now.
![127 0 0 1_5500_setup-mobileversion_project1 html](https://user-images.githubusercontent.com/39240820/171233300-725d8800-a92d-4e7f-b047-b7fcfe83b0b6.png)